manual/manual/refman/exten.etex

```diff
@@ -1593,12 +1593,11 @@ Some attributes are understood by the type-checker:
  the same effect as the "-w" command-line option, in the scope between
  the attribute and the end of the current
  signature/structure/object/object type.  The attribute can also be
- used on an expression,
- in which case its scope is limited to that expression.
+ attached to any kind of syntactic item which support attributes
+ (such as an expression, or a type expression)
+ in which case its scope is limited to that item.
  Note that it is not well-defined which scope is used for a specific
  warning.  This is implementation dependant and can change between versions.
```

Comment:
I think it would be a good idea to mention here that some warnings are not affected by such attributes. In particular, lexical warnings (latin 1 identifiers, comment end and start, illegal backslash, and unattached documentation comment) are not affected. Maybe more problematic, the flambda warning 59 is simultaneously not affected and can yield false positive.

Reply:
Done.